packages/web/graphql/post/posts.graphql

```diff
@@ -9,6 +9,7 @@ query posts(
   $hasInteracted: Boolean
   $authoredOnly: Boolean!
```

Comment:
Since we're restoring `authorId` we should get rid of `authoredOnly` shouldn't we?